help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
-   [curate and shape them up](https://github.com/datagrok-ai/public/blob/master/help/develop/release-history.md#dev-build-08936)
-   by hand with the lists of highlights, major items, improvements, bug fixes and cross-linking to relevant
-   Community Forum posts, examples and sources. Therefore, Notes become a powerful source of full knowledge
-   about what's coming in the recent Datagrok update.
-
-3. Highly diversed set of [JS API Samples](https://public.datagrok.ai/js) right at hand. Use the built-in
-   [JS Fiddler](https://public.datagrok.ai/js) inside Datagrok to fast-prototype interfaces and functions
-   which will later become parts of your packages. The sources for these API Samples are versioned
-   [here](https://github.com/datagrok-ai/public/tree/master/packages/ApiSamples).
-   
-4. [Datagrok Public repository](https://github.com/datagrok-ai/public). We are passionate about making
-   more parts of our work open-source, as well as on involving other developers into expanding the platform.
-   The results of this you may find on our [public github](https://github.com/datagrok-ai/public).  
-   
-   Not only this is a great source of really many API use examples, but also a repository of fine-crafted
-   patterns, such as a [Chembl data repository browser](), and plugins, such as
-   [custom cell renderers](https://github.com/datagrok-ai/public/blob/master/packages/Chem/src/rdkit_cell_renderer.js),
-   [custom viewers](https://github.com/datagrok-ai/public/tree/master/packages/Viewers),
-   [custom filters](https://github.com/datagrok-ai/public/blob/master/packages/Widgets/src/filters/radio_button_filter.js), and so on. Also find there our packages for domains such as
-   [Digital Signal Processing](https://github.com/datagrok-ai/public/tree/master/packages/DSP), [Biosignals](https://github.com/datagrok-ai/public/tree/master/packages/BioSignals), [Cheminformatics](),
-   or [Natural Language Processing](https://github.com/datagrok-ai/public/tree/master/packages/NLP).
-
-5. We host regular Datagrok User Meetings. Ask Andrew or Dan for a Zoom link
-   (for example, by a private message at [Community Forum](https://community.datagrok.ai/)).
+Check out the following spaces for more information about Datagrok:
 
-## Datagrok Video Walkthrough
+1. [Community Forum]. This is where we post updates about the Datagrok platform and ecosystem. We recommend that you
+   subscribe at least to the following three topics on the forum:
+   * [JavaScript API Updates]
+   * [Visualization-related Updates]
+   * [Cheminformatics Updates]
 
-Let's get it started!
+2. [Datagrok Release History] &mdash; this page aggregates the release notes. For every new Datagrok build, the release notes
+   are automatically generated. However, for the major releases, we curate our release notes, which you can see on
+   [our GitHub page]. There, we detail the highlights, major items, improvements, and bug fixes in the releases. We also
```

Comment:
@AndSviat the link target is missing.

Reply:
The links aren't missing. They are added to the bottom of the page:
![not missing any links](https://user-images.githubusercontent.com/21691607/127444486-3e9c7e67-bf8f-4c24-b6a6-1568180a9b7c.png)